python/paddle/distributed/auto_parallel/static/parallelizer_v2.py

```diff
@@ -354,6 +355,17 @@ def _apply_post_optimization(
             )
             params_grads = self._pass_context.get_attr("params_grads")
 
+        mp_async_allreduce_in_backward = os.getenv(
+            "FLAGS_mp_async_allreduce_in_backward"
```

Comment:
could use config like:
config["use_sharding"] = self._strategy.sharding.enable
for switch